media-tv/kodi/kodi-9999.ebuild

```diff
@@ -152,131 +148,78 @@ In some cases Kodi needs to access multicast addresses.
-	cp "${DISTDIR}/libdvdcss-${LIBDVDCSS_COMMIT}.tar.gz" "${S}/tools/depends/target/libdvdcss/libdvdcss-master.tar.gz" || die
-	cp "${DISTDIR}/libdvdread-${LIBDVDREAD_COMMIT}.tar.gz" "${S}/tools/depends/target/libdvdread/libdvdread-master.tar.gz" || die
-	cp "${DISTDIR}/libdvdnav-${LIBDVDNAV_COMMIT}.tar.gz" "${S}/tools/depends/target/libdvdnav/libdvdnav-master.tar.gz" || die
 }
 
 src_prepare() {
 	default
 
-	# some dirs ship generated autotools, some dont
-	multijob_init
-	local d dirs=(
-		tools/depends/native/TexturePacker/src/configure
-		$(printf 'f:\n\t@echo $(BOOTSTRAP_TARGETS)\ninclude bootstrap.mk\n' | emake -f - f)
-	)
-	for d in "${dirs[@]}" ; do
-		[[ -e ${d} ]] && continue
-		pushd ${d/%configure/.} >/dev/null || die
-		AT_NOELIBTOOLIZE="yes" AT_TOPLEVEL_EAUTORECONF="yes" \
-		multijob_child_init eautoreconf
-		popd >/dev/null || die
-	done
-	multijob_finish
-	elibtoolize
-
-	# Cross-compiler support
-	# We need JsonSchemaBuilder and TexturePacker binaries for the host system
-	# Later we need libsquish for the target system
-	if tc-is-cross-compiler ; then
-		mkdir "${WORKDIR}"/${CBUILD} || die
-		pushd "${WORKDIR}"/${CBUILD} >/dev/null || die
-		einfo "Building host tools"
-		cp -a "${S}"/{tools,xbmc} ./ || die
-		local tool tools=( JsonSchemaBuilder )
-		use texturepacker && tools+=( TexturePacker )
-		for tool in "${tools[@]}" ; do
-			tc-env_build emake -C tools/depends/native/$tool
-			mkdir "${S}"/tools/depends/native/$tool/bin || die
-			ln -s "${WORKDIR}"/${CBUILD}/tools/depends/native/$tool/bin/$tool \
-				"${S}"/tools/depends/native/$tool/bin/$tool || die
-		done
-		popd >/dev/null || die
-
-		emake -f codegenerator.mk
-
-		# Binary kodi.bin links against libsquish,
-		# so we need libsquish compiled for the target system
-		emake -C tools/depends/native/libsquish-native/ CXX=$(tc-getCXX)
-	elif [[ ${PV} == 9999 ]] || use java ; then #558798
-		tc-env_build emake -f codegenerator.mk
-	fi
-
-	# Disable internal func checks as our USE/DEPEND
-	# stuff handles this just fine already #408395
-	export ac_cv_lib_avcodec_ff_vdpau_vc1_decode_picture=yes
-
-	# Fix the final version string showing as "exported"
-	# instead of the SVN revision number.
-	export HAVE_GIT=no GIT_REV=${EGIT_VERSION:-exported}
-
 	# avoid long delays when powerkit isn't running #348580
 	sed -i \
 		-e '/dbus_connection_send_with_reply_and_block/s:-1:3000:' \
 		xbmc/linux/*.cpp || die
-
-	# Tweak autotool timestamps to avoid regeneration
-	find . -type f -exec touch -r configure {} + || die
 }
 
 src_configure() {
-	# Disable documentation generation
-	export ac_cv_path_LATEX=no
-	# Avoid help2man
-	export HELP2MAN=$(type -P help2man || echo true)
-	# No configure flage for this #403561
-	export ac_cv_lib_bluetooth_hci_devid=$(usex bluetooth)
-	# Requiring java is asine #434662
-	[[ ${PV} != 9999 ]] && export ac_cv_path_JAVA_EXE=$(which $(usex java java true))
+	local CMAKE_BUILD_TYPE=$(usex debug Debug Release)
```

Comment:
Try and avoid this like the plague. `CMakeLists.txt` should be written such that it doesn't care about our `CMAKE_BUILD_TYPE=Gentoo`.